pages/common/llvm-as.md

```diff
@@ -0,0 +1,20 @@
+# llvm-as
+
+> LLVM Intermediate Representation (`.ll`) to Bitcode (`.bc`) assembler.
+> More information: <https://llvm.org/docs/CommandGuide/llvm-as.html>.
+
+- Assemble an IR file:
+
+`llvm-as -o {{path/to/out.bc}} {{path/to/source.ll}}`
+
+- Assemble an IR file and include a module hash in the produced Bitcode file:
+
+`llvm-as --module-hash -o {{path/to/out.bc}} {{path/to/source.ll}}`
+
+- Read IR from `stdin` and assemble it:
+
+`llvm-as -o {{path/to/out.bc}}`
+
+- Assemble an IR file to `stdout` (might mess up your terminal):
```

Comment:
Not sure if printing Bitcode to `stdout` is a good idea. Does this example have any use case?

Reply:
You might want to pipe it to `opt` or something like that.

Reply:
Then could we turn this into a more useful example? That's actually the purpose of tldr.

Reply:
In this case I don't think it would be a particular problem to include an example command that would be useful. You mention `opt` @pixelcmtd - perhaps in this instance we could include this as an explicit example?

Reply:
If you only like it with a program that it pipes to, I'd instead remove it, because then it doesn't show anything new anymore.

An example of how to make a pipe like `clang | llvm-as | opt | llc` should be in an `llvm` page.